modules/KalturaSupport/components/chapters/chapters.js

```diff
@@ -1311,10 +1324,12 @@
             };
 
 			var mediaBoxes = this.getMediaListDomElements();
-			mediaBoxes.on('mousedown mouseup mouseout', function(){
-				this.blur();
-			})
-			.on('keyup', $.proxy(this.onItemKey , this));
+			mediaBoxes
+				.on('mousedown mouseup mouseout', function(){
+					this.blur();
+				})
+				.on('keydown', $.proxy(this.handleHomeEndKeys, this))
```

Comment:
why separate keydown and keyup? does it matter if we use any of them? if not lets consolidate to one and reuse `onItemKey` for all

Reply:
player handle 'keyup' before plugin and makes seekToBeginning and seekToEnd

Reply:
For this case there is `onDisableKeyboardBinding` event to disable the main player keyboard bindings and `onEnableKeyboardBinding` to bring them back.

Reply:
I think for this scenario, Yuval created this:
https://github.com/kaltura/playkit-js-ui/blob/master/docs/with-keyboard-event.md
@OrenMe @yairans ?

Reply:
cool,
I'll make the changes with `onDisableKeyboardBinding` and `onEnableKeyboardBinding`

Reply:
As the solution @OrenMe suggested turns out to be much more complex (due to other `onEnableKeyboardBinding` and `onDisableKeyboardBinding` which are fired irrelevantly) we will still stick with @semarche-kaltura solution